queries/countries.rq

```diff
@@ -21,6 +21,7 @@ WHERE {
     (wd:Q55 'Netherlands' 'netherlands' 'Current content includes ministries, courts, provincies and municipalities.')
     (wd:Q664 'New Zealand' 'new-zealand' 'Current content includes district health boards, wānangas, state-owned enterprises, regional councils and territorial authorities.')
     (wd:Q33 'Finland' 'finland' 'Current content includes regions.')
+    (wd:Q212 'Ukraine' 'ukraine' 'Current content includes ministries and first level administrative units')
```

Comment:
Does the first level administrative units have a name other than "first level administrative units"?

Reply:
They have three different names in Wikidata. Not sure if we rather want to go with "oblasts, cities with special status and region".